fah_xchem/analysis/diffnet.py

```diff
@@ -0,0 +1,311 @@
+import logging
+from math import log
+from typing import List, Optional
+
+import networkx as nx
+import numpy as np
+
+from ..schema import (
+    Compound,
+    CompoundAnalysis,
+    CompoundMicrostate,
+    Microstate,
+    MicrostateAnalysis,
+    PointEstimate,
+    TransformationAnalysis,
+)
+from .exceptions import AnalysisError, ConfigurationError, InsufficientDataError
+
+
+def pIC50_to_DG(pIC50: float, s_conc: float = 375e-9, Km: float = 40e-6) -> float:
+    """
+    Converts IC50 (in M units) to DG
+    Parameters
+    ----------
+    pIC50 : float
+        pIC50
+    s_conc : float, default=375E-9
+        Substrate concentration in M
+    Km : float, default=40E-6
+        Substrate concentration for half-maximal enzyme activity
+    Returns
+    -------
+    type
```

Comment:
Returns the dimensionless free energy (in kT)

Reply:
Thanks, fixed in 05f17db.